Makefile

```
@@ -1,132 +1,191 @@
+
+.PHONY: license
+license: bin/wwhrd vendor
+	./hack/license.sh
+
+.PHONY: verify-license
+verify-license: bin/wwhrd vendor
+	./hack/verify-license.sh
+
+.PHONY: tidy
 tidy:
 	go mod tidy -v
 
-build: ${ARTIFACT}
-
-${ARTIFACT}: ${SOURCES}
-	CGO_ENABLED=0 go build ${GOMOD} -i -installsuffix cgo ${LDFLAGS} -o ${ARTIFACT} ./cmd/manager/main.go
-
-build-plugin: ${ARTIFACT_PLUGIN}
-
-${ARTIFACT_PLUGIN}: ${SOURCES}
-	CGO_ENABLED=0 go build -i -installsuffix cgo ${LDFLAGS} -o ${ARTIFACT_PLUGIN} ./cmd/${ARTIFACT_PLUGIN}/main.go
-
-container:
-	bin/operator-sdk build $(PREFIX):$(TAG)
-    ifeq ($(KINDPUSH), true)
-	 kind load docker-image $(PREFIX):$(TAG)
-    endif
-
-container-ci:
-	docker build -t $(PREFIX):$(TAG) --build-arg "TAG=$(TAG)" .
-
-test:
-	./go.test.sh
-
-e2e:
-	hack/generate-e2etest-manifest.sh
-	bin/operator-sdk test local ./test/e2e --global-manifest ./test/e2e/global-manifest.yaml --go-test-flags '-v' --image $(PREFIX):$(TAG)
-
-push: container
-	docker push $(PREFIX):$(TAG)
-
-clean:
-	rm -f ${ARTIFACT}
-	rm -rf ./bin
-
-validate: bin/golangci-lint bin/wwhrd
-	bin/golangci-lint run ./...
-	hack/verify-license.sh
-
-generate: generate-k8s generate-crds patch-crds
-
-generate-k8s: bin/operator-sdk
-	bin/operator-sdk generate k8s
-
-generate-crds: bin/operator-sdk
-	rm -f ./deploy/crds/*.yaml
-	bin/operator-sdk generate crds --crd-version=v1beta1
-	mv ./deploy/crds/*.yaml ./deploy/crds/v1beta1/
-	./bin/operator-sdk generate crds --crd-version=v1
-	mv ./deploy/crds/*.yaml ./deploy/crds/v1/
-
-patch-crds: bin/yq generate-crds
-	hack/patch-crds.sh
-
-generate-openapi: bin/openapi-gen bin/client-gen bin/informer-gen bin/lister-gen
-	bin/openapi-gen --logtostderr=true -o "" -i ./pkg/apis/datadoghq/v1alpha1 -O zz_generated.openapi -p ./pkg/apis/datadoghq/v1alpha1 -h ./hack/boilerplate.go.txt -r "-"
-	hack/generate-groups.sh client,lister,informer \
-  github.com/datadog/extendeddaemonset/pkg/generated github.com/datadog/extendeddaemonset/pkg/apis datadoghq:v1alpha1 \
-  --go-header-file ./hack/boilerplate.go.txt
-
-generate-olm: bin/operator-sdk
-	bin/operator-sdk generate packagemanifests --version $(VERSION:v%=%) --update-crds --interactive=false
-
-pre-release: bin/yq
-	hack/pre-release.sh $(VERSION) $(RELEASE_CANDIDATE)
+.PHONY: vendor
+vendor:
+	go mod vendor
 
-CRDS = $(wildcard deploy/crds/*_crd.yaml)
-local-load: $(CRDS)
-		for f in $^; do kubectl apply -f $$f; done
-		kubectl apply -f deploy/
-		kubectl delete pod -l name=${PROJECT_NAME}
+kubectl-eds: fmt vet lint
+	go build -ldflags '${LDFLAGS}' -o bin/kubectl-eds ./cmd/kubectl-eds/main.go
 
-$(filter %.yaml,$(files)): %.yaml: %yaml
-	kubectl apply -f $@
+bin/kubebuilder:
+	./hack/install-kubebuilder.sh 2.3.1
 
-install-tools: bin/golangci-lint bin/operator-sdk bin/yq
+bin/openapi-gen:
+	go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
 
 bin/yq:
-	hack/install-yq.sh
+	./hack/install-yq.sh 3.3.0
 
 bin/golangci-lint:
 	hack/golangci-lint.sh v1.18.0
```

to be consistent with other make target.

```suggestion
	./hack/golangci-lint.sh v1.18.0
```